pkg/loki/modules.go

```diff
@@ -34,7 +34,8 @@ import (
 	"github.com/grafana/loki/pkg/querier"
 	"github.com/grafana/loki/pkg/querier/queryrange"
 	loki_storage "github.com/grafana/loki/pkg/storage"
-	"github.com/grafana/loki/pkg/storage/stores/local"
+	"github.com/grafana/loki/pkg/storage/stores/shipper"
+	shipper_uploads "github.com/grafana/loki/pkg/storage/stores/shipper/uploads"
```

Comment:
is there a collision that requires this? (I didn't look super close just aw it in the diff)